src/components/card/z-card-dictionary-info/index.tsx

```diff
@@ -0,0 +1,109 @@
+import { Component, Prop, h, Event, EventEmitter } from "@stencil/core";
+import { DictionaryData } from "../../../beans";
+
+/**
+ * @slot  - content
+ */
+@Component({
+  tag: "z-card-dictionary-info",
+  styleUrl: "styles.css",
+  shadow: true
+})
+export class ZCardDictionaryInfo {
+  /** dictionary info */
+  @Prop() data: string | DictionaryData;
+
+  private cardData: DictionaryData;
+
+  /** flip card to front */
+  @Event() closeInfo: EventEmitter;
+  emitCloseInfo() {
```

Comment:
l'evento dovrebbe chiamarsi `toggleInfo` se viene lanciato in entrambe le direzioni (apertura e chiusura)